### PR TITLE
Add GH Labels

### PR DESCRIPTION
**Pull request by upodroid**
# Changes

I need some labels to prioritise some work and few area specific labels for test-infra

/cc @dprotaso @kvmware

### CODE REVIEW

File: label_sync/labels.yaml [hunk 1]
```diff
@@ -265,6 +265,41 @@ default:
...
+      name: priority/important-longterm
+      target: both
+      prowPlugin: label
+      addedBy: anyone
+    - color: eb6420
+      description: Must be staffed and worked on either currently, or very soon, ideally in time for the next release.
+      name: priority/important-soon
+      target: both
+      prowPlugin: label
+      addedBy: anyone
```
Comment (kvmware) [inline label_sync/labels.yaml:301]: These seem to be what one would use the project board for which sorta captures these states already.

File: label_sync/labels.yaml [hunk 2]
```diff
@@ -276,6 +311,31 @@ repos:
...
+    - color: 0052cc
+      name: area/tools
+      description: Issues or PRs related to internal tools found at /tools
+      target: issues
+      addedBy: anyone
+    - color: 0052cc
+      name: area/images
+      description: Issues or PRs related to testing docker images found at /images
+      target: issues
+      addedBy: anyone
```
Comment (kvmware) [inline label_sync/labels.yaml:338]: :+1:

File: label_sync/labels.yaml (hunk 1 above)
Comment (upodroid) [reply label_sync/labels.yaml:301]: It doesn't today.

https://github.com/orgs/knative/projects/40/views/1

For example, we have 10 items ready to work but they don't have the priority labels so everyone doesn't know which one is important, work that we have deffered

Comment (kvmware) [reply label_sync/labels.yaml:301]: image
Notice you can even title the priority field as you like

Comment (upodroid) [reply label_sync/labels.yaml:301]: I didn't see that column. Cool

**Comment by dprotaso** [comment]
@upodroid do you need this PR anymore (since you can add custom columns to GitHub Projects )

**Comment by upodroid** [comment]
I still think we need it. The columns aren't as good as labels as the fields are write-in, the column doesn't appear in the issues list or in the specific issue's page.

**Pull request merged**